katdal/visdatav4.py

```diff
@@ -142,14 +142,18 @@ class VisibilityDataV4(DataSet):
         while the keyword 'all' means all available products will be applied.
         *NB* In future the default will probably change to 'all'.
         *NB* This is still very much an experimental feature...
+    gaincal_fluxes : dict mapping string to float, optional
```

Comment:
How about `flux_gaincals`? Which maybe implies that its a single flux per gaincal that the user can provide.

But I'm happy with this either way:)

Reply:
Or `gaincal_flux`? Looks more like `measured_flux` and the word `gaincal` should already make you think "which gaincal"?